content/blog/posts/this-is-how-we-watchdog.md

```diff
@@ -0,0 +1,40 @@
+---
+title: This is how we watchdog
+date: 2020-12-21
+author: Eva Ruth Moravec
+---
+We frequently mention that we “watchdog” the data that we work with as much as possible, but I thought I’d peel that back a bit. What do TJI’s oversight efforts look like? 
+
+Each month, I get new data from the Texas Office of the Attorney General reflecting the previous month's reported deaths in custody and shootings of and by law enforcement officers. I add this new data to TJI's main data sets. Overnight, bots look for changes to the data sets, which are then pushed through our processing pipeline. 
```

Comment:
How about we provide a link to "the Texas Office of the Attorney General"?

Reply:
great idea!